### DISCUSSION TITLE
Alert if Highlight found

### DISCUSSION THREAD

**Discussion by petvo78** [variar/klogg#457] [Ideas]
Hi, would be great implement an "alert". Whenever a Highlight is triggered, add possibility to make a Tray-icon message or calling an external application.

**Comment by variar** [comment]
Hi, could you describe a usage scenario for this? It looks like you want to open a file, enable follow mode, select a highlight pattern and then after a new line that matches the highlight appears in the file klogg should do something to notify user.

**Comment by petvo78** [comment]
Hi! Yes exactly. 

Like 

- "send email",
-  "Microsoft tray icon alert (toast)" 
- or trigger an external app. 

App "LogExpert" uses a solution like this: https://i.imgur.com/xmW773H.png

**Comment by variar** [comment]
Ok, I see how that  could be useful. I don’t want to add heavy functions like sending an email. Triggering external app/script should be enough for most cases. Or we can add a plug-in system (I saw something like that in one of the forks), and then the notification plugins could be developed separately.
